twitter.py

```diff
@@ -120,16 +120,24 @@ def count_followers(user: dict, from_date: datetime, to_date: datetime) -> dict:
     The function counts all followers for the particular user and for each day within the specified period.
     It returns a dictionary that contains a list of dates and a list of number of followers for each day.
     """
-    cur.execute('''SELECT followed_at, COUNT(*) FROM follows 
-        WHERE followed_at BETWEEN %s AND %s and whom = %s
-        GROUP BY followed_at ORDER BY followed_at;''', (from_date, to_date, user['id']))
-
-    number_of_followers = cur.fetchall()
-    date_when, followers_per_day = zip(*number_of_followers)
+    followers_per_day = []
+    date_when = []
+    while from_date <= to_date:
```

Comment:
this does (for now) the same thing as other function - in case there's a gap for any day, it will put there zero.